node/network/collator-protocol/src/validator_side/mod.rs

```diff
@@ -662,6 +662,14 @@ async fn fetch_collation<Context>(
 
 	if state.peer_data.get(&peer_id).map_or(false, |d| d.has_advertised(&relay_parent)) {
 		request_collation(ctx, state, relay_parent, para_id, peer_id, tx).await;
+	} else {
```

Comment:
This does also trigger if there is no peer data for that peer.

Reply:
Yes, I think this message shoul be on the `warn!` level as it is quite unlikely.